plugins/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/RuntimeProjectDescriptor.xtend

```diff
@@ -284,12 +284,10 @@ class RuntimeProjectDescriptor extends TestedProjectDescriptor {
 							}
 							
 							// outline API
-							fragment = adapter.FragmentAdapter {
-								fragment = outline.OutlineTreeProviderFragment auto-inject {}
-							}
-							fragment = adapter.FragmentAdapter {
-								fragment = outline.QuickOutlineFragment auto-inject {}
-							}
+							fragment = ui.outline.OutlineTreeProviderFragment2 {}
+							
+							// quick outline menu contribution
```

Comment:
Please also adjust the expectations in CLIWizardIntegrationTest. 

See https://xtext-builds.itemis.de/jenkins/job/xtext-runtime-tests-master-latest/640/#showFailuresLink